Dockerfile

```diff
@@ -25,12 +26,13 @@ RUN cd contracts && npm install
 USER root
 
 # Build the Go binary in a separate stage
-FROM golang:1.22 as builder
+FROM golang:1.22 AS builder
 WORKDIR /app
 COPY go.mod go.sum ./
 RUN go mod download
 COPY . .
-RUN go build -v -o masa-node ./cmd/masa-node
+# Inject the latest git tag version into the build
+RUN go build -v -o masa-node -ldflags "-X 'github.com/masa-finance/masa-oracle/pkg/config.Version=$(git describe --tags --abbrev=0)'" ./cmd/masa-node
```

Comment:
My suggestion here would be to do something around: 

```
git describe --always --tags || echo "dev" 
```

Reply:
also.. can't we just call `make` here too? so we avoid to spread out the logic

Reply:
Call make here to accomplish what?

Reply:
to build the binary - as you added https://github.com/masa-finance/masa-oracle/pull/438/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R5

Reply:
@mudler I'm sorry, I'm not sure I understand what you would like changed.

Reply:
instead of replicating the `go build ... ` command inside the Dockerfile, can we call `make build` instead from the `Dockerfile` to generate the binary? in this way we don't have to replicate the `go build` logic across the codebase, but keep it tight to the Makefile (and using only one place to build/handle git tag and versioning)

Reply:
ahh, so reference the makefile within the docker container so we have a single point where this logic is being modified?

Reply:
yep!